.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Default ownership to "Product Direction"
+/content/en/contribute/medic @medic/product-leaders
+
+# Allow product management people to do their stuff in the main area for it
+/content/en/contribute/medic/product-development-process @medic/product-management
```

Comment:
I'm not a fan of CODEOWNERS files because it goes against the spirit of openness, restricts improvements, blocks improvements if people are on holiday, and tends to request reviews from many people when one would do.

We've tried them in the past and removed them because of these failings. Why do we think it'll work better this time?